.readthedocs.yml

```diff
@@ -18,6 +18,8 @@ build:
   os: ubuntu-22.04
   tools:
     python: "3.8"
+  apt_packages:
+    - graphviz
```

Comment:
Would using the [graphviz](https://www.sphinx-doc.org/en/master/usage/extensions/graphviz.html) plugin work for this?